.github/actions/android-ci/action.yml

```diff
@@ -75,3 +75,16 @@ runs:
       shell: bash
       working-directory: ${{github.workspace}}/build
       run: cmake --build . --parallel 2 --target install
+
+    - name: Upload native library
+      uses: actions/upload-artifact@v4
+      with:
+        path: ./build/lib/libf3d-java.so
+        name: f3d-android-native-${{inputs.arch}}
+
+    - name: Upload Java package
+      if: inputs.arch == 'arm64-v8a' # avoid duplication
```

Comment:
what duplication ?

Reply:
F3d.jar is the same regardless of the architecture. I don't want to pollute artifacts with duplicate archives

Reply:
ok got it